crates/sui/src/unit_tests/keytool_tests.rs

```diff
@@ -501,7 +502,7 @@ async fn test_sign_command() -> Result<(), anyhow::Error> {
 
     // Sign an intent message for the transaction data without intent passed in, so default is used.
     KeyToolCommand::Sign {
-        address: *sender,
+        address: KeyIdentity::Address(*sender),
```

Comment:
Should we have some unit tests that use `KeyIdentity::Alias`?

Reply:
Definitely, let me add those!